lib/galaxy_test/api/test_tools_upload.py

```diff
@@ -223,6 +223,13 @@ def test_image_upload_auto(self):
             tiff_metadata = self._upload_and_get_details(fh, file_type="auto")
         self.assertEqual(tiff_metadata["file_ext"], "tiff")
 
+    @skip_without_datatype("ome.tiff")
+    def test_ometiff_upload_sniff(self):
+        tiff_path = TestDataResolver().get_filename("1.ome.tiff")
+        with open(tiff_path, "rb") as fh:
+            tiff_metadata = self._upload_and_get_details(fh, file_type="auto")
+        self.assertEqual(tiff_metadata["file_ext"], "ome.tiff")
+
```

Comment:
Please remove this and move  test-data/1.ome.tiff to lib/galaxy/datatypes/test, this will then be automatically tested without having to write a test.

Reply:
Done.